roles/gateway_preupgrade_analyzer/README.md

```diff
@@ -1,23 +1,64 @@
-Pre-Upgrade-Upgrader
+Preupgrade Analyzer
 ======================
 
-This role will inform which customizations on the Gateway will be migrated automatically, and which ones customer will need to migrate themself.
+This role will generate pre-upgrade analyzer report for all servers listed in hosts file under [gateway] section. 
```

Comment:
It would be nice to mention somewhere that the reports will be collected and stored on the Ansible controller.